src/test/test_parsers.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from src.base import HTMLParser, HTMLParserInput, HTMLParserOutput
+from src.newsplease import NewsPleaseParser
+from src.readability import ReadabilityParser
+from src.combined import CombinedParser
+
+
+@pytest.mark.parametrize(
+    "url",
+    [
+        # "https://www.legislation.gov.au/Details/F2020L00552",
```

Comment:
This line is commented out because the NewsPleaseParser fails on this URL with the following error. This error only occurs when running the test in Docker.

``` bash
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.legislation.gov.au', port=443): Max retries exceeded with url: /Details/F2020L00552 (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1129)')))

/usr/local/lib/python3.9/site-packages/requests/adapters.py:563: SSLError
```

So far I've tried the following unsuccessfully:

1. Adding `RUN apt-get install ca-certificates` to the top of the Dockerfile
2. `pip install --upgrade certifi`
3. Changing python versions used in the Docker image (3.9.5, 3.8)

Reply:
This works for me in my REPL:
```
>>> requests.get("https://www.legislation.gov.au")
<Response [200]>
>>> requests.get("https://www.legislation.gov.au/Details/F2020L00552")
<Response [200]>

```
What about you?

Reply:
I found out the issue was only with the `CombinedParser` because there was no exception handling around `requests.get()` as there is in the other parsers. I've updated the `CombinedParser` to include this exception handling and the tests to ensure that the empty response is not returned for each of the test cases.

Side note: at the moment when a website can't be fetched we just return an empty response. Do we want to return the exception as part of the output?

Reply:
From some Slack debugging it seems like this might be an issue with the legislation.gov.au domain. I've created [another issue](https://linear.app/climate-policy-radar/issue/DSCI-99/assess-size-of-ssl-handshake-error-bug) to see if it is so we can merge this and get on with building a pipeline 🚴‍♂️